scrapy_zyte_api/providers.py

```diff
@@ -234,6 +241,18 @@ async def __call__(  # noqa: C901
                 item = AnnotatedInstance(Geolocation(), cls.__metadata__)  # type: ignore[attr-defined]
                 results.append(item)
                 continue
+            if cls_stripped is Actions and is_typing_annotated(cls):
+                actions_result: Optional[List[_ActionResult]]
+                if "actions" in api_response.raw_api_response:
+                    actions_result = [
+                        _ActionResult(**action_result)
+                        for action_result in api_response.raw_api_response["actions"]
+                    ]
+                else:
+                    actions_result = None
+                item = AnnotatedInstance(Actions(actions_result), cls.__metadata__)  # type: ignore[attr-defined]
```

Comment:
The variable name ("item") looks wrong; this applies to Geolocation dependency as well.